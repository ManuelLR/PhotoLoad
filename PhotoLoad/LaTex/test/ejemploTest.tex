% !TeX spellcheck = es_ES
\documentclass{scrartcl}
%\documentclass{book}
\usepackage[spanish]{babel}
\usepackage[utf8]{inputenc}
\usepackage{enumerate}
\usepackage{graphicx}
\usepackage{color}
\usepackage{float}
\usepackage{multicol}
\usepackage{lscape}
\usepackage{fancyhdr}
\usepackage{tabularx}
\usepackage[hidelinks,colorlinks=true, linkcolor=black]{hyperref} %hyperref para hacer los links del indice y usar \url{URL} y \href{URL}{text}y hidelinks para que no se rodeen con una caja de color los enlaces. Más información en: http://en.wikibooks.org/wiki/LaTeX/Hyperlinks
\pagestyle{fancy}
\begin{document}


\subsection{prueba 1}

\begin{tabularx}{14cm}{|c|X|}
	\hline \textbf{ID} & prueba1 \\ 
	\hline \textbf{Descripción} & Esta es una descripcion muy muy muy larga que se ajustará automáticamente a la celda
	
	y esto es lo que pasa si quiero metertle más espacio \\	 
	\hline  \textbf{Entrada}		& A escribir \\ 
	\hline  \textbf{Salida esperada}			& Po JIIII \\
	\hline  \textbf{Resultado}			& Fracaso \\
	\hline 
\end{tabularx} 

\subsection{prueba 2}

\begin{tabularx}{14cm}{|c|X|}
	\hline \textbf{ID} & prueba2 \\ 
	\hline \textbf{Descripción} & Esta es una descripcion muy muy muy larga que se ajustará automáticamente a la celda
	
	y esto es lo que pasa si quiero metertle más espacio \\	 
	\hline  \textbf{Entrada}		& A escribir \\ 
	\hline  \textbf{Salida esperada}			& Po JIIII \\
	\hline  \textbf{Resultado}			& Fracaso \\
	\hline 
\end{tabularx} 
\end{document}