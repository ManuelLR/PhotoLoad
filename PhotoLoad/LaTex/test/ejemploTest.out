\BOOKMARK [2][-]{subsection.0.1}{prueba 1}{}% 1
\BOOKMARK [2][-]{subsection.0.2}{prueba 2}{}% 2
